### ISSUE TITLE
Erroneous null value in generated edit_* notebook

### ISSUE THREAD

**Issue by AntoineRondelet** [fivetran/great_expectations#3742]
**Describe the bug**

After generating a first set of expectation from one of my dataset using
```console
great_expectations --v3-api suite new
```
and selecting option `3` to create the expectation suite automatically with the profiler, I then decided to refine the expectation suite.
To do this, I ran:
```console
great_expectations --v3-api suite edit <my_expectation_suite>
```
I first selected the interactive mode to do a round of modifications. I then used the manual mode to finalize a few things. However, while editing the expectation suite in manual mode, I got a notebook with the following code excerpt in the code cells:
```python
expectation_configuration = ExpectationConfiguration(**{
  "ge_cloud_id": null,
  "meta": {},
  "kwargs": {
    "column": <my-column-name>
  },
  "expectation_type": "expect_column_values_to_not_be_null"
})
suite.add_expectation(expectation_configuration=expectation_configuration)
```
Running this cell throws the following error:
```console
NameError: name 'null' is not defined
```
which is to be expected, since no quotes are surrounding `null`. I suspect this comes from an erroneous loading of JSON file to generate the Python code of the notebook. `null` in JSON should become `None` in Python.

**To Reproduce**

I haven't tried with different `great_expectations` contexts, but I suspect that reproducing should be feasible by following the steps I mentioned above, i.e.
1. Given a dataset, generate an automated set of initial expectations
2. Edit your expectations in manual mode
3. Run the notebook
4. The cells containing expectation code should throw the error mentioned above

**Expected behavior**

Replace `null` (note the missing surrounding quotes) by `None` in Python code, including Python cells in generated notebooks.

**Environment (please complete the following information):**
 - Operating System: Linux - Ubuntu
 - Great Expectations Version: 0.13.43

**Additional context**

**Comment by AntoineRondelet** [comment]
Note: I quickly checked this behavior against the tutorial (https://docs.greatexpectations.io/docs/tutorials/getting_started/intro) and the notebook `edit_getting_started_expectation_suite_taxi.demo` generated when running:
```console
great_expectations --v3-api suite edit getting_started_expectation_suite_taxi.demo
```
also contains a `null` value for the `"ge_cloud_id"` key in the dictionary passed as input to `ExpectationConfiguration` causing the error mentioned above.

**Comment by cdkini** [comment]
Hey @AntoineRondelet! Thanks for bringing up this issue.

We'll review the information you've provided above and confer internally before determining next steps. Appreciate your patience!

**Comment by cdkini** [comment]
@AntoineRondelet the changes you're requesting should be solved as part of #3782. Thanks so much!

I'll close this issue but please let me know if anything else pops up.
